pkg/apis/core/types.go

```diff
@@ -5788,3 +5796,470 @@ type PortStatus struct {
 	// +kubebuilder:validation:MaxLength=316
 	Error *string
 }
+
+// PodResourceClaim references exactly one ResourceClaim, either by name or
```

Comment:
```suggestion
// PodResourceClaim references a ResourceClaim, either by name or
```

just simpler wording

Reply:
Actually, the description is confusing, because it's mostly describing ClaimSource, rather than this object.

Reply:
True. This comes from a time when ClaimSource didn't exist yet. Updated.